metacmd/cmds.go

```diff
@@ -49,7 +50,20 @@ func init() {
 				"? ": {"show help on special variables", "variables"},
 			},
 			Process: func(p *Params) error {
-				Listing(p.Handler.IO().Stdout())
+				name, err := p.Get(false)
+				if err != nil {
+					return err
+				}
+				switch name {
+				default:
+					Listing(p.Handler.IO().Stdout())
+				case "commands":
+					Listing(p.Handler.IO().Stdout())
+				case "options":
+					kingpin.Usage()
```

Comment:
This is some bad tight coupling. It's fine for this, but let's put in a `// FIXME: decouple`.

Reply:
I agree, I don't have an idea how to do it better right now. Added a comment.